tests/integration/component.go

```diff
@@ -385,9 +385,12 @@ func componentTests(args ...string) {
 			})
 
 			It("should not fail when --context is not set", func() {
-				// Was failing due to https://github.com/openshift/odo/issues/1969
+				binaryFilePath := filepath.Join(commonVar.Context, "sb.jar")
+				if runtime.GOOS == "darwin" {
+					binaryFilePath = filepath.Join("/private", binaryFilePath)
+				}
 				helper.Cmd("odo", append(args, "create", "--s2i", "java:8", cmpName, "--project",
-					commonVar.Project, "--binary", filepath.Join(commonVar.Context, "sb.jar"))...).ShouldPass()
+					commonVar.Project, "--binary", binaryFilePath...).ShouldPass()
```

Comment:
Missed a closing bracket there.
```suggestion
			helper.Cmd("odo", append(args, "create", "--s2i", "java:8", cmpName, "--project",
					commonVar.Project, "--binary", binaryFilePath)...).ShouldPass()
```